tests/x11/validate_vnc_from_client.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate that after remote installation, the VNC connection
+# between client and host, is available without the use of vnc password.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use mmapi;
+use y2lan_restart_common "close_xterm";
+
+# Returns the first leased IP, in /var/lib/dhcp/db/dhcpd.leases file.
+sub get_lease_ipaddr {
+    my $lease_ip;
+    my $dhcp_leases_file = script_output("cat /var/lib/dhcp/db/dhcpd.leases");
+    for my $line (split /\n/, $dhcp_leases_file) {
+        if ($line =~ /^lease\s+(?<lease_ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})/) {
+            return $+{lease_ip};
```

Comment:
Correct me if i am wrong. 
- you do not need definition in L25 
- there is no check for `lease_ip`. In case of empty or corrupted *dhcpd.leases*, it will return null

Reply:
right, make sense to die if not found at the end of the method.

Reply:
I added an assert_script_run before, as it seems more readable.

Reply:
that helps, but if the content does not match is the same issue, best thing is to store the capture, die if empty or return it.
But anyway I think we can fix that value as I mentioned in other comment.

Reply:
Done